dotcom-rendering/index.d.ts

```diff
@@ -679,6 +679,14 @@ type DCRSupportingContent = {
 	format: ArticleFormat;
 };
 
+type FEEnrichedContent = {
+	embedHtml?: string;
+	embedCss?: string;
+	embedJs?: string;
+};
+
+type DCREnrichedContent = FEEnrichedContent;
+
```

Comment:
```suggestion
type FEEnrichedContent = {
	embedHtml?: string;
	embedCss?: string;
	embedJs?: string;
};

type DCREnrichedContent = {
	embedHtml?: string;
	embedCss?: string;
};

```
This might be a nice way to convey our intentions. What do you think?

Reply:
Also, just a thought, but what are your feelings on landing on just 'snap' as the naming convention and leaving 'enriched' out of DCR? E.g.: Replace `DCREnrichedContent ` with `DCRSnapType`. Up to you 🤷‍♂️